### PR TITLE
Previous name

### PR DESCRIPTION
**Pull request by sjpadgett**
fix select2 bootstrap theme

#### Short description of what this resolves:
Eventually the previous name history for ONC.
We haven't gulp-glob installed so don't know how select2-BS4 ever worked. Still, it now works for all themes except light and for the life of me don't see why! Getting really frustrated as every time I try to do any development, I spend hours fixing stuff in core!
I can't be the only one that notices these errors/missteps. I beg everyone to step up and fix em when you see em! This goes for vendors. I know of only two vendors that have contributed in last year and it's getting tiresome....

### CODE REVIEW

**Comment by sjpadgett** [comment]
Gave up as to why select2 styles not working on light theme. Even tho the compiled style_light.css has the styles for bootstrap 4 theme, only the container see the styles applied. Assume somewhere they are overridden but I can't find!
Include from config works so, it doesn't bug too much anymore.:)

**Comment by sjpadgett** [comment]
@adunsulag 
I didn't map uuid for patient history new columns as didn't see the need however, if you need let me know.
Let me know if services mod's cover what you need.
Previous names management is all handled from the multiselect in edit patient. Add and delete with no edits. Currently names don't show in tab display. Still figuring that out.
If need today let me know or pull from my branch.

**Comment by stephenwaite** [comment]
awesome job, it fixes a pesky bug too:

when someone would have a secondary insurance and needed to choose something from the secondary medicare type dropdown

temporary

File: interface/patient_file/summary/demographics_full.php [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 // Session pid must be right or bad things can happen when demographics are saved!
 //
-$set_pid = isset($_GET["set_pid"]) ? $_GET["set_pid"] : ($_GET["pid"] ?? null);
+$set_pid = $_GET["set_pid"] ?? ($_GET["pid"] ?? null);
```
Comment (stephenwaite) [inline interface/patient_file/summary/demographics_full.php:27]: cool-cat

File: config/config.yaml [hunk 2]
```diff
@@ -174,11 +174,13 @@ assets:
...
-        basePath: '%assets_static_relative%/select2/dist/'
-        script: js/select2.full.min.js
+        basePath: '%assets_static_relative%/'
+        script: 'select2/dist/js/select2.full.min.js'
         link:
-            - css/select2.min.css
+            - 'select2/dist/css/select2.min.css'
+            - '@ttskch/select2-bootstrap4-theme/dist/select2-bootstrap4.min.css'
             # BS4 theme moved to main theme assets to allow fluidity
+            # Imported scss doesn't work on light theme. Left import for overrides.
```
Comment (bradymiller) [inline config/config.yaml:183]: @tywrenn , Not sure if you have time at the moment, but looks like we are having some issues with the imported select scss on the light theme.

File: interface/patient_file/summary/demographics_full.php (hunk 1 above)
Comment (bradymiller) [reply interface/patient_file/summary/demographics_full.php:27]: 118382341-53d9d600-b5c2-11eb-8b7e-220bbab8468a

File: interface/themes/core.scss [hunk 3]
```diff
@@ -9,7 +9,7 @@
  */
 
 // Import Select2 BS4 Theme
-@import "../../public/assets/@ttskch/select2-bootstrap4-theme/src/layout";
+@import "../../public/assets/@ttskch/select2-bootstrap4-theme/src/select2-bootstrap4";
```
Comment (sjpadgett) [inline interface/themes/core.scss:12]: @bradymiller This seems to have fixed the issue although I've rebuilt several times in past with no joy till tried again this a.m and works so, beats me. Maybe a windows thing.
I'm going to revert config change and style the selection remove 'x' so easier for mobile devices.
Then I guess we'll see on demos.

**Comment by sjpadgett** [comment]
Manilla theme:
image

Light theme
image

Dark:
image

image

**Comment by sjpadgett** [comment]
landed here with style for select2. still don't understand why some builds fail. hoping just a Jerry thing...
bringing in soon
image

**Comment by sjpadgett** [comment]
I haven't tried on mobile but I think i've given enough room to remove items from select2 selection with touch. We need to add a few classes to our containers throughout core for IOS but, I don't think many if any.
At some point as soon as I get a chance, i'll convert the data types for list with add over to new specialty forms to be consistent.
Thanks everyone.

**Comment by adunsulag** [comment]
Thanks @sjpadgett for doing this!  I looked through the code and didn't see any problems.  That was a lot of work!

**Comment by sjpadgett** [comment]
You're welcome. You guys are worth it!:) 
I wanted to see if I could get a different pattern working for new types of uses for plugin forms in LBF. Converting the list with add data type will be a good next test.

**Pull request merged**